connector-packager/connector_packager/xsd_validator.py

```diff
@@ -69,6 +69,8 @@ def validate_all_xml(files_list: List[ConnectorFile], folder_path: Path) -> bool
         else:
             xml_violations_found += 1
 
+        check_file_content(file_to_test, path_to_file)
```

Comment:
Do we want to fail the validation if they use DefaultSQLDialect? If so, maybe return a bool from this method and, if it   returns false fail the connector.

Reply:
I think we don't want to fail the validation since a warning is preferred here, not an error.